translations/strings.yaml

```diff
@@ -244,6 +244,9 @@ whatsNewReleaseNotes:
     value: Mozilla VPN is now integrated with the Multi-Account Containers add-on for more privacy and security.
     comment: “Multi-Account Containers” is the name of the add-on and should not be translated.
   multiAccountContainersDescription: You can now use different Mozilla VPN locations in multiple tabs simultaneously.
+  connectionInfoDisplayName: Test your speed
+  connectionInfoDescription: You can now run speed tests in Mozilla VPN! Select the performance icon in the top left corner of your screen to test your connection speed while using VPN.
```

Comment:
We currently don't support RTL languages, but this might be something considering for the future (the icon will be on the right).